.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
-repos:
-  - repo: https://github.com/psf/black
-    rev: 19.10b0
-    hooks:
-      - id: black
+- repo: https://github.com/psf/black
```

Comment:
We can do this in a different PR and see why it was pinned to the specific version